Core/Transaction.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Core
 {
-	public class Transaction : IComparable<Transaction>
+	public struct Transaction : IComparable<Transaction>
```

Comment:
Codacy found an issue: [Implement 'IEquatable<T>' in value type 'Transaction'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)